daemon/internal/builder-next/worker/worker.go

```diff
@@ -14,11 +14,11 @@ import (
 	cerrdefs "github.com/containerd/errdefs"
 	"github.com/containerd/log"
 	"github.com/containerd/platforms"
-	imageadapter "github.com/docker/docker/builder/builder-next/adapters/containerimage"
-	mobyexporter "github.com/docker/docker/builder/builder-next/exporter"
+	imageadapter "github.com/docker/docker/daemon/internal/builder-next/adapters/containerimage"
+	mobyexporter "github.com/docker/docker/daemon/internal/builder-next/exporter"
+	"github.com/docker/docker/daemon/internal/mod"
```

Comment:
FWIW; `mod` is only used in builder-next/worker; we could consider moving it internal to that;
https://github.com/moby/moby/blob/291872e086c8baada74aa8e9c92b1e4048aceb0d/builder/builder-next/worker/worker.go